src/extension/src/scripts/rnw-dependencies.ps1

```diff
@@ -323,10 +405,10 @@ if ($NeedsRerun -ne 0) {
     #WTS: When replacing this file review changes to check no throw is sent on failure
     #Write-Error "Some dependencies are not met. Re-run with -Install to install them.";
     #throw;
-    Write-Error "Some dependencies are not met.";
+    Write-Error "Some dependencies are not met. Re-run with -Install to install them.";
     return;
 } else {
     Write-Output "All mandatory requirements met";
+    $Tags | Out-File $MarkerFile
     return;
-}
-
+}
```

Comment:
EOF?

Reply:
Good catch! Thanks.

Reply:
Resolved in 5b70dab395f9ac158553b0b411efa8d6858bf2a6